probatus/binning/binning.py

```diff
@@ -437,9 +436,8 @@ def tree_bins(x, y, inf_edges, tree):
 
         bin_count = len(index)
 
-        tpf = TreePathFinder(tree)
-        boundaries = [bound["min"] for bound in tpf.get_boundaries().values()]
-        boundaries += [tpf.get_boundaries()[leaves[-1]]["max"]]
```

Comment:
In the previous code the boundary was added only on the right side, but in the new code it is added to both left and right side. Could you double check if it is correct?

Reply:
I did, it's correct. I think it's some sort of max of the leaves or something